docs/installation.rst

```diff
@@ -14,23 +14,25 @@ and ``pip`` in these instructions.
 
 Getting Cedar-solve
 --------------
-Cedar-solve is not available on PyPI (the Python Package Index) yet. Instead you need to provide
-a link to or download the GitHub source code.
+Cedar-solve is available on PyPI (the Python Package Index). You may install it by::
 
-Use PIP to download and install
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-The easiest method is to let PIP download from GitHub and install. This will set up all
-dependencies and make the package usable from anywhere.::
+    pip install cedar-solve
 
-    pip install git+https://github.com/smroid/cedar-solve.git
+You can test that it works by invoking the database generator command::
 
-You can test that it works by running the example provided in the GitHub repo.
+    tetra3-gen-db --help
```

Comment:
you can't run the example when installing from a wheel, we don't include that directory or the images inside the wheel.

Reply:
if they install it from pypi its assumed they are integrating it into their project in some way.
they can download the source or clone the git repo to run the tests and the example.

an alternative to including the example would be a minimal example described in the documentation as a  tutorial.